pandas/tests/window/test_pairwise.py

```diff
@@ -189,3 +189,33 @@ def test_corr_freq_memory_error(self):
         result = s.rolling("12H").corr(s)
         expected = Series([np.nan] * 5, index=date_range("2020", periods=5))
         tm.assert_series_equal(result, expected)
+
+    def test_cov_mulittindex(self):
+        # GH 34440
+
+        # create multiindexed DataFrame
+        columns = MultiIndex.from_product([["a", "b"], ["x", "y"], [0, 1]])
+        index = range(3)
+        df = DataFrame(
+            np.random.normal(size=(len(index), len(columns))),
+            index=index,
+            columns=columns,
+        )
+
+        # flatten index then compute covariance
+        df_fi = df.copy()
+        df_fi.columns = ["".join(str(c) for c in col) for col in df_fi.columns.values]
+        cov_fi = df_fi.ewm(alpha=0.1).cov()
+        cov_fi.index = [
+            "".join(str(symbol) for symbol in row_label) for row_label in cov_fi.index
+        ]
+
+        # compute covariance matrix then flatten its multtindex
+        df_mi = df.copy()
```

Comment:
is feasible to hard code the results here (you can use a fixed seed). in a constructed frame?

Reply:
can also use fixed inputs rather than random nums.

Reply:
yes I can do that